Gem/Assets/Types/AddTokenInput.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Primitives
+import Settings
+
+struct AddTokenInput {
+    let availableChains: [Chain]
+
+    var chain: Chain?
+    var address: String?
+
+    init(wallet: Wallet) {
```

Comment:
can we pass `availableChains` in init? and calculate `availableChains` in the view model